VAMobile/src/translations/en/common.json

```diff
@@ -733,12 +733,11 @@
   "letters.benefitService.disabledDueToService": "You {{areOrNot}} considered to be totally and permanently disabled solely due to your service-connected disabilities.",
   "letters.benefitService.disabledDueToServiceA11yHint": "Select whether to include service connected permanent disability information in your letter",
   "letters.benefitService.dischargeType": "Discharge type",
-  "letters.benefitService.effectiveDate": "The effective date of the last change to your current award was {{date}}.",
+  "letters.benefitService.effectiveDateInvalid": "invalid date",
```

Comment:
I noticed when this is used, the toggle text reads "... the last change to your current payment was invalid date." This might be a content question for Misty, but do we want to update this to `an invalid date` to sound more correct, or should we add some emphasis to the `invalid date` text to show that this would normally be a populated value that happens to be `null`

Reply:
@mistymg Can you let us know which you would prefer?

Reply:
Good question!

Reply:
@mistymg @theodur I had to end up going with the first option after looking at the code again. That particular section uses a SimpleListItemObj component and all it takes in is text which doesn't allow for customization. Without having to greatly change a lot of our list components this wouldn't be possible. There is already a code upkeep ticket to unravel our list components to make this possible in the future and we can revisit it then.

Reply:
Ahh that's a bummer, I agree that the second option would have been ideal. Forgot these were simple list objects under the hood